CMakeLists.txt

```diff
@@ -27,6 +27,17 @@ if(IS_CI_BUILD)
     add_compile_options(-DCI_BUILD=1)
 endif()
 
+if(EXISTS ${CMAKE_SOURCE_DIR}/.git/HEAD)
+    exec_program(
+        "git"
+        ${CMAKE_SOURCE_DIR}
+        ARGS "rev-parse --short=8 HEAD"
+        OUTPUT_VARIABLE MBGL_VERSION_REV )
+    add_definitions(-DMBGL_VERSION_REV="${MBGL_VERSION_REV}")
```

Comment:
Seeing that we are removing the global version symbol, so we'll either have to reintroduce that, or list all the files that use the define. Also noticing that Android is not using this define yet.